User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,79 @@
+# Survey questions
+
+## Readme
+
+This survey aims at evaluating users’ experiences with ESMValTool and
+discovering users’ needs.  It is meant for the users of ESMValTool in all levels
+of expertise. Some criteria to design the questions:
+
+- It does not take more than 10 (?) minutes
+- It does not have overlapping questions
+- It does not include questions with yes/no answers
+- It does not collect personal data, but it has some profiling questions
+- It covers different aspects of ESMValTool like application, usage, documentation and community
+
+The survey can be implemented in a survey app like surveymonkey that does not
+collect personal data. The owner is “user engagement team”. They will
+edit/run/evaluate the survey and communicate the results to other teams if
+needed.
+
+## Survey content
+
+Hi! We in [ESMValGroup](https://www.esmvaltool.org/), would love to hear about
+your research and experiences using ESMValTool. Here, there are a few questions
+which help to create your story. Your answers will help us to improve ESMValTool
+and hence enhance the science that you conduct. All responses are recorded
+anonymously. Let’s begin:
+
+**Describe your research/case study and challenges:**
+
+1. Can you describe your research/case study?
```

Comment:
From a survey analysis perspective, what would be learnt by asking this question? Would knowing the details about a user's specific research be useful? Would it be enough to know whether they use ESMValTool for atmosphere, ocean, etc diagnostics, or for real time model evaluation, or for creating plots for IPCC reports, etc.? Would it be worth adding multiple choice answers for this?